File: Daily Challanges/2026-03-March/2026-03-22.py
ROAST_LEVEL = {
    "'": 1,
    "-": 2,
    ".": 3
}
def detect_roast(beans):
    points = 0
    for bean in beans:
        points += ROAST_LEVEL[bean]
    avg_points = points/len(beans)
    if avg_points < 1.75:
        return "Light"
    elif avg_points <= 2.5:
        return "Medium"
    else:
        return "Dark"
    return beans

print(detect_roast("''-''''''-'-''--''''"))
print(detect_roast(".--.-..-......----.'"))

File: Daily Challanges/2026-03-March/2026-03-22.js
const ROAST_LEVEL = {
  "'": 1,
  "-": 2,
  ".": 3,
};

function detectRoast(beans) {
  let points = 0;

  for (const bean of beans) {
    points += ROAST_LEVEL[bean];
  }

  const avgPoints = points / beans.length;

  if (avgPoints < 1.75) {
    return "Light";
  } else if (avgPoints <= 2.5) {
    return "Medium";
  } else {
    return "Dark";
  }
}

console.log(detectRoast("''-''''''-'-''--''''"));
console.log(detectRoast(".--.-..-......----.'"));
